.githooks/pre-commit

```diff
@@ -9,7 +9,11 @@
 # ln -s ../../.githooks/pre-commit .git/hooks/pre-commit
 # ```
 
-set -euxo pipefail
+set -euo pipefail
+
+script_dir=$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")
+
+tf_modules=$("$script_dir/../scripts/collect-modules.sh")
```

Comment:
I would put the CI-related scripts in the `.github/scripts`,
and keep `./scripts` is for customer-used scripts

Reply:
Sure, moved the script